### PR TITLE
add support for yts.mx tracker

### PR DESCRIPTION
**Pull request by NecroKote**
obviously, adds support for "yts.mx" tracker.

(I personally see no use in adding this tracker, as there are basically no 'updates' as in other forum-based ones, other than adding torrents from this tracker via Telegram bot. But adding it anyway)

Should satisfy #66

### CODE REVIEW

File: torrt/trackers/ytsmx.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        return movie_id
+
+    def _get_movie_details(self, movie_id: str) -> dict:
+        """returns dict with movie info by calling API"""
+
+        response = self.get_response(API_BASE + f"movie_details.json?movie_id={movie_id}")
+        if not response:
+            raise YtsmxTrackerException("API didn't respond")
+
+        response: Response
```
Comment (idlesign) [inline torrt/trackers/ytsmx.py:79]: Is it necessary? You may want to  type hint base `.get_response()` instead.

**Review by idlesign — CHANGES_REQUESTED** [review]
Thank you. Almost ready to merge.
I have a few comments though.

Also please add the tracker mentioning in README.

File: torrt/trackers/ytsmx.py (hunk 1 above)
Comment (NecroKote) [reply torrt/trackers/ytsmx.py:79]: Yea, in this case, I think it's necessary, as during `.get_response` it's still of type of `Optional[Response]` (eg. `None | Response`).
So I think this line clarifies the type we are working with below. Explicitly.

File: torrt/trackers/ytsmx.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        movie_id = movie_info_tag.attrs['data-movie-id']
+        if not movie_id.isdigit():
+            raise YtsmxTrackerException("movie-id is not a digit. Markup is changed")
+
+        return movie_id
+
+    def _get_movie_details(self, movie_id: str) -> dict:
+        """returns dict with movie info by calling API"""
+
+        response = self.get_response(API_BASE + f"movie_details.json?movie_id={movie_id}")
```
[the remark this answers is not in the document]
Comment (NecroKote) [reply torrt/trackers/ytsmx.py]: fixed

File: tests/trackers/test_ytsmx.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
+from typing import Dict
```
[the remark this answers is not in the document]
Comment (NecroKote) [reply tests/trackers/test_ytsmx.py:1]: fixed. most string literals covered by single quotes (except for a few f-strings, containing single quotes inside)

**Review by idlesign — APPROVED** [review]

**Comment by idlesign** [comment]
Thank you. Merged.

**Pull request merged**